### PR TITLE
MoneroKon D-1 Spam Mitigation Transcription

### PR DESCRIPTION
**Pull request by lh1008**
Hey @erciccione, 

Transcription for: 
**Spam Mitigation and Blockchain Size Control in Privacy Supporting Permissionless Blockchains**
_Francisco Cabañas, The Monero Project_ 

Author @smeemon 

https://www.monerooutreach.org/monero-konferenco/josh-s.php 

Taiga Instance 213 
Review is being worked by @JOhNKmus 
https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/213

Thank you @smeemon, great work!

### CODE REVIEW

**Comment by erciccione** [comment]
@lh1008 Did you include @JOhNKmus 's suggestions? is this ready to go?

**Comment by smeemon** [comment]
I checked 'approve' on the changes in github. I'll look around a bit
more on github and taiga when I have a bit more time later this week,
but if there's anything more I need to do on this one let me know.
Thanks

**Comment by lh1008** [comment]
Hey @erciccione,

Yes, I did include @JOhNKmus suggestions and also made him as co-author in the last commit. This is ready to go.

>:)

File: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Spam_Mitigation_The_Challenge_of_Private_and_Permissionless.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+The one fee you can look at, somewhat, if you want to increase it, is the lowest fee. That is doable. I mean, you can even take it out entirely, and just set the minimum fee just slightly below the normal fee. That's a possibility, you're going to get about a five times gain just out of that on the maintenance. But I would say the real answer lies in addressing the motive, and the motive is attacking the privacy side of the coin, in that particular case.
+
```
Comment (erciccione) [inline monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Spam_Mitigation_The_Challenge_of_Private_and_Permissionless.txt]: Please remove all the following newlines. Leaving just one would be fine

Comment (lh1008) [reply monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Spam_Mitigation_The_Challenge_of_Private_and_Permissionless.txt]: Okay, I will add something else before pushing. Will do this in a moment.

**Comment by lh1008** [comment]
Okay @erciccione, 

This should be ready to go. Changed the files name, did a final review, found a few things to change and everything is a go. I also added the sites missing text in the final commit.

>:)

**Pull request merged**